lib/command/src/Obelisk/Command.hs

```diff
@@ -45,8 +47,12 @@ data Args = Args
   }
   deriving Show
 
-args :: Parser Args
-args = Args <$> noHandoff <*> verbose <*> obCommand
+newtype ArgsConfig = ArgsConfig
+  { _argsConfig_enableVmBuilderByDefault :: Bool
+  }
```

Comment:
Can you explain the need for this? I wonder how `--enable-vm-builder` is essentially different from other flags (like `-v`).

Reply:
This changes the default behavior of the flag parsing. I.e. on Darwin you don't need to pass this flag and it will still be enabled.